doc/nrf/releases_and_maturity/software_maturity.rst

```diff
@@ -170,27 +170,150 @@ Protocol support
-+--------------------+-----------+-----------+-----------+-----------+-----------+---------------------+---------------------+--------------+--------------+---------------------+---------------------+---------------------+
-| **Sidewalk**       | --        | --        | --        | --        | --        | Supported           | Supported           | --           | Experimental | --                  | --                  | --                  |
-+--------------------+-----------+-----------+-----------+-----------+-----------+---------------------+---------------------+--------------+--------------+---------------------+---------------------+---------------------+
-| **Thread**         | --        | --        | --        | --        | Supported | Supported           | Supported           | Experimental | Experimental | --                  | --                  | --                  |
-+--------------------+-----------+-----------+-----------+-----------+-----------+---------------------+---------------------+--------------+--------------+---------------------+---------------------+---------------------+
-| **Wi-Fi**          | --        | --        | --        | --        | --        | Supported\ :sup:`1` | Supported\ :sup:`2` | Experimental | Experimental | Supported\ :sup:`1` | Supported\ :sup:`1` | Supported\ :sup:`1` |
-+--------------------+-----------+-----------+-----------+-----------+-----------+---------------------+---------------------+--------------+--------------+---------------------+---------------------+---------------------+
-| **Zigbee**         | --        | --        | --        | --        | Supported | Supported           | Supported           | --           | --           | --                  | --                  | --                  |
-+--------------------+-----------+-----------+-----------+-----------+-----------+---------------------+---------------------+--------------+--------------+---------------------+---------------------+---------------------+
+.. list-table::
+   :widths: auto
+   :header-rows: 1
+
+   * -
+     - nRF52810
+     - nRF52811
+     - nRF52820
+     - nRF52832
+     - nRF52833
+     - nRF52840
+     - nRF5340
+     - nRF54H20
+     - nRF54L15
+     - nRF9131
+     - nRF9151
+     - nRF9160
+     - nRF9161
+   * - **Bluetooth**
+     - Supported
+     - Supported
+     - Supported
+     - Supported
+     - Supported
+     - Supported
+     - Supported
+     - Experimental
+     - Experimental
+     - --
+     - --
+     - --
+     - --
+   * - **Bluetooth Mesh**
+     - --
+     - --
+     - --
+     - Supported
+     - Supported
+     - Supported
+     - Supported
+     - --
+     - Experimental
+     - --
+     - --
+     - --
+     - --
+   * - **DECT NR+ PHY**
+     - --
+     - --
+     - --
+     - --
+     - --
+     - --
+     - --
+     - --
+     - --
+     - Experimental
+     - Experimental
+     - --
+     - Experimental
+   * - **LTE**
```

Comment:
list tables may be difficult to track (finding which line corresponds to which column)

Reply:
But if it is grid table, doesn't it make it more difficult to make a change? You need to make space alignments for the entire table for a change in one of the values/entries.